common/modules/farm/rewards/src/rewards.rs

```diff
@@ -38,7 +38,7 @@ pub trait RewardsModule:
         if current_block_nonce > last_reward_nonce {
             let to_mint = self.calculate_per_block_rewards(current_block_nonce, last_reward_nonce);
             if to_mint != 0 {
-                mint_function(token_id, &to_mint);
+                mint_function(self, token_id, &to_mint);
```

Comment:
we need to change - as no more mint here.

Reply:
Same as above.